packit/api.py

```diff
@@ -2176,6 +2177,12 @@ def run_osh_build(
         else:
             cmd = ["osh-cli", "mock-build", str(srpm_path)]
 
+        if csmock_args is None:
+            csmock_args = self.package_config.csmock_args
```

Comment:
This is a user controlled input that is passed to the service, and later run as a shell command. As I understand, it is escaped by the `run_command()` function, so this should not cause arbitrary command execution. I need someone from the Packit team to verify it.

Reply:
Yes, verified [here](https://openscanhub.fedoraproject.org/task/8907/log/stdout.log)